tests/general/IndentationBearTest.py

```diff
@@ -130,7 +130,7 @@ def test_blank_lines(self):
                       "\tIndents even after blank line}")
         invalid_file = ("{ trying indent",
                         "\n",
-                        "should've Indented after blank line}")
+                        "should ave Indented after blank line}")
```

Comment:
`should have` ;)

But why exactly can't we have that? (I will take a look at the algorithm itself later when I'm home from class, as that one takes some effort to review). But you said you didn't break anything... and honestly it shouldn't be a problem to use single quotes in double-quote-strings and the other way around. I mean, python allows that just fine so it shouldn't lead to errors or problems.